app/main.py

```diff
@@ -609,16 +621,24 @@ def manage_login_modal(n_clicks, username, password, login_storage, current_cent
             form_feedback.append(html.P("Il semble qu'il manque votre nom d'utilisateur et/ou votre mot de passe."))
 
             # The login modal remains open; other outputs are updated with arbitrary values
-            return True, {"login": "no"}, form_feedback, {"center": [10, 10], "zoom": 3}, {"display": "none"}
+            return [
+                True,
+                {"login": "no"},
+                form_feedback,
+                {"center": [10, 10], "zoom": 3},
+                {"display": "none"},
+                dash.no_update,
+            ]
 
         else:
             # This is the route of the API that we are going to use for the credential check
             try:
-                Client(cfg.API_URL, username, password)
+                client = Client(cfg.API_URL, username, password)
```

Comment:
the variable isn't used afterwards, no need to keep it

Reply:
It is used in the `return` statement just below (line 666).

Reply:
oh my bad!